dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/keras/optimizers/Optimizers.scala

```diff
@@ -4,8 +4,20 @@ import me.shadaj.scalapy.tensorflow.api.Tensor
 import me.shadaj.scalapy.tensorflow.keras.optimizers.{Optimizers => PyOptimizers}
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.{PythonModule, PythonType}
 import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonOption._
+import me.shadaj.scalapy.tensorflow.api.scalaUtils.PythonEnum
 import scala.language.implicitConversions
 
+enum OptimizerNames (override private[api] val v: String) extends PythonEnum(v){
```

Comment:
In Layers.scala, enum names are singular, here it's plural... pls make consistent (I'd rather go for singular, unless some Python/TF convention prefers the latter)

Reply:
I think plural is the norm here. for example: https://www.tensorflow.org/api_docs/python/tf/keras/metrics
Anyway I will make it consistent.